mitodl/ol-concourse-github-issues
# cpatti add trigger tomestone

### What are the relevant tickets?

Fixes #52 

### Description (What does it do?)
When a closed Github concourse workflow issue is 'consumed' by the resource and kicks off a Concourse build, we change the title to begin with the prefix [CONSUMED #<JOB NUMBER>].

### How can this be tested?

Successfully tested using a [test pipeline](https://cicd.odl.mit.edu/teams/infrastructure/pipelines/issue-create-test). You can see an example of a properly 'tombstoned' [consumed issue](https://github.com/mitodl/concourse-workflow/issues/1282).
